test/protocol/test/textDocument/documentSymbol/documentSymbol.test.ts

```diff
@@ -93,12 +93,12 @@ describe('[hardhat] documentSymbol', () => {
         },
         selectionRange: {
```

Comment:
all of these results regressed from selecting the specific identifier, to entire lines (or multiple lines for this example). Not sure I understand why?

Reply:
I switched the approach a bit. Before, we were using the identifier's range as the selection range (so clicking on the outline for e.g. a contract definition, would select the contract name). But now it will select the entire definition. I'm using the definition rule node's textRange for this. Should I make any adjustment/revert it to the previous approach?

Reply:
Thanks for explaining. In that case, your new logic is more correct.
I checked the TypeScript LSP, and they also select the whole definition range, not just the identifier.